sample/kotlin/build.gradle.kts

```diff
@@ -139,7 +139,13 @@ android {
     }
 }
 
+repositories{
+    mavenLocal()
+}
+
 dependencies {
+    implementation("io.opentelemetry:opentelemetry-api:1.4.0")
```

Comment:
it is already a part of `dd-sdk-android-trace` with `api` visibility though, so not needed here.

Reply:
yes it will not be like that in the implementation anyway, this is just me playing with things here. In the implementation we will only provide the `opentelemetry` at compile time and the clients will add the dependency on their end. That's also the way APM does.